.gitignore

```diff
@@ -5,7 +5,9 @@ local_settings.py
 *.egg-info
 .treerc
 .coverage
+.idea
 .tox
+.pytest_cache
```

Comment:
Nice, that's a recent improvement: https://github.com/pytest-dev/pytest/issues/3138 .  Used to be `.cache`, which made it unclear at a glance where it was coming from.